proto/firecracker.proto

```diff
@@ -92,6 +95,40 @@ message GetVMMetadataResponse {
     string Metadata = 1;
 }
 
+message UpdateBalloonRequest {
+    string VMID = 1;
+    int64 AmountMb = 2;
+}
+
+message UpdateBalloonStatsRequest {
+    string VMID = 1;
+    int64 StatsPollingIntervals = 2;
+}
+
+message GetBalloonStatsRequest {
+    string VMID = 1;
+}
+
+message GetBalloonStatsResponse {
+    int64 ActualMb = 1; // Actual amount of memory (in MB) the device is holding.
```

Comment:
nit indentation.

Reply:
Yeah, I noticed these. But I have no idea how to format firecracker.proto. Is there a tool we used like "gofmt -w" to help format protobuf? I saw there was some on github wrote by others but not sure if I can use it for this project.